### PR TITLE
Setup server directory in Vim script

### PR DESCRIPTION
**Pull request by thinca**
All installer script do same things:  Remove the previous server and create a new directory.

This has 2 problems:
1. Reduced maintainability.
2. Changing install directory is too hard.

By this PR, all installer scripts assume invoked at the installing path.

I'll send an other PR to be configurable the installing server path if this PR is accepted.

### CODE REVIEW

**Comment by mattn** [comment]
Totally LGTM, Please wait my confirmations.

File: installer/install-bash-language-server.cmd [hunk 1]
```diff
@@ -1,5 +1,3 @@
 @echo off
 
-cd /d %~dp0
-
-call npm_install bash-language-server bash-language-server
+call "%~dp0\npm_install" bash-language-server bash-language-server
```
Comment (mattn) [inline installer/install-bash-language-server.cmd]: This was my lazy thing. But could you please add `.cmd` for this `npm_install ?

Comment (thinca) [reply installer/install-bash-language-server.cmd]: I added `.cmd` for all `call "%~dp0\xxx"`.

**Comment by mattn** [comment]
Thank you

**Pull request merged**